src/main/java/core/Main.java

```diff
@@ -0,0 +1,16 @@
+package core;
+
+import core.db.Storage;
+import core.model.Product;
+
+public class Main {
+    public static void main(String[] args) {
+        Product iphone = new Product("Iphone 11", 1200);
+        Product iphoneX = new Product("Iphone X", 1000);
+
+        Storage.addProduct(iphone);
+        Storage.addProduct(iphoneX);
+
+        Storage.products.forEach(System.out::println);
```

Comment:
I agree with Artem
we cannot directly manipulate Storage, only **through service**

open the link that Artem shared and follow that example
you must have `private static Injector injector` field
and `ProductService productService` local variable, which methods you will test